rules/do-you-know-how-modern-stateless-auth-works/rule.md

```diff
@@ -0,0 +1,67 @@
+---
+type: rule
+title: Do you know how Modern Stateless Authentication works?
+uri: modern-stateless-authentication
+authors:
+  - title: Dhruv Mathur
+    url: https://www.ssw.com.au/people/dhruv-mathur/
+  - title: Gordon Beeming
+    url: https://www.ssw.com.au/people/gordon-beeming/
+  - title: Matt Goldman
+    url: https://www.ssw.com.au/people/matt-goldman/
+created: 2024-03-06T13:42:55.753Z
+guid: fa645fa6-4054-448b-9b4b-d60a66422f1f
+---
+
+Modern stateless authentication, leveraging JSON web tokens (JWT), offers a scalable solution for managing user sessions without server-side storage. The OIDC framework, building on OAuth 2.0, standardizes interactions between authentication services and users. JWTs, containing a header, payload, and signature, ensure data integrity and authentication validity. Issued by trusted identity providers, these tokens facilitate user identification in applications, moving away from traditional session-based methods. This approach enhances scalability and user management, aligning with current web development trends.
+
+`youtube: https://www.youtube.com/embed/P2CPd9ynFLg`
+**Figure: Why JWT is popular? (5min)**
```

Comment:
```suggestion
**Video: Why JWT is popular? (5 min)**
```

1. Videos have a Video prefix - https://www.ssw.com.au/rules/add-useful-and-concise-figure-captions/#tip-1-use-prefixes
2. Time length has a space by `min` - https://www.ssw.com.au/rules/right-format-to-write-videos-time-length/

Reply:
Also change to Why is JWT popular?